# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436993810 14400
#      Wed Jul 15 16:56:50 2015 -0400
# Node ID 5e50f25f245a3def79055199d1403281872ebb5f
# Parent  c3dab5d8e6ab8775021ae0eb8774ef87827890b6
* move-if-change: Write message to stderr.

diff --git a/build-aux/move-if-change b/build-aux/move-if-change
--- a/build-aux/move-if-change
+++ b/build-aux/move-if-change
@@ -1,15 +1,15 @@
 #!/bin/sh
 #
 # Like mv $1 $2, but if the files are the same, just delete $1.
 # Status is 0 if $2 is changed, 1 otherwise.
 
 if test -r $2; then
   if cmp $1 $2 > /dev/null; then
-    echo $2 is unchanged
+    echo $2 is unchanged 1>&2
     rm -f $1
   else
     mv -f $1 $2
   fi
 else
   mv -f $1 $2
 fi
